app/lib/katello/resources/candlepin/product.rb

```diff
@@ -4,7 +4,7 @@ module Candlepin
       class Product < CandlepinResource
         class << self
           def all(owner_label, included = [])
-            products = JSON.parse(Candlepin::CandlepinResource.get(path(owner_label) + "?#{included_list(included)}", self.default_headers).body)
+            products = JSON.parse(Candlepin::CandlepinResource.get(path(owner_label) + "?active=include&#{included_list(included)}", self.default_headers).body)
```

Comment:
```suggestion
           url = path(owner_label) + "?active=include" + "&#{included_list(included)}"
            products = JSON.parse(Candlepin::CandlepinResource.get(url, self.default_headers).body)
```

slightly more readable imp